tenpy/linalg/symmetries.py

```diff
@@ -543,15 +542,9 @@ def __init__(self, factors: list[Symmetry]):
             braiding_style=max((f.braiding_style for f in flat_factors), key=lambda style: style.value),
             trivial_sector=np.concatenate([f.trivial_sector for f in flat_factors]),
             group_name=' ⨉ '.join(f.group_name for f in flat_factors),
-            num_sectors=np.prod([symm.num_sectors for symm in flat_factors]),
+            num_sectors= np.prod([symm.num_sectors for symm in flat_factors]), ####PROBLEM HERE WITH INT
             descriptive_name=descriptive_name
         )
-        self.has_fusion_tensor = all(f.has_fusion_tensor for f in flat_factors)
-        dtypes = [f.fusion_tensor_dtype for f in flat_factors]
-        if None in dtypes:
-            self.fusion_tensor_dtype = None
-        else:
-            self.fusion_tensor_dtype = Dtype.common(*dtypes)
```

Comment:
please dont remove this.

Reply:
I did not actively remove this, maybe my branch was not updated??